appcelerator-archive/amp
# Elastic 5.3

# content

- update elasticsearch version to 5.3.0
- new image for kibana (5.3.0)
- fix #825 (etcd 3.1.5)

# how to test

```
ampmake clean build build-cli-wrapper
amp start
for image in amp-agent ampbeat amplifier; do docker tag appcelerator/${image}:local localhost:5000/appcelerator/${image}:local ; docker push localhost:5000/appcelerator/${image}:local ; done
docker run -it --rm --network=hostnet -v $PWD/stacks:/stacks docker --host=m1 stack deploy -c /stacks/monitoring.stack.yml monitoring
docker run -it --rm --network=hostnet -v $PWD/stacks:/stacks docker --host=m1 stack deploy -c /stacks/amplifier.stack.yml amplifier
docker exec m1 docker service ls # amplifier, etcd, elasticsearch and kibana should be up
# you should be able to connect to http://localhost:5601
amp stop
```
